test/0.8.9/unstructured-storage.test.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.24 <0.9.0;
+
+import "forge-std/Test.sol";
+import { UnstructuredStorage } from "contracts/0.8.9/lib/UnstructuredStorage.sol";
+
+contract ExposedUnstructuredStorage {
+    function _getStorageBool(bytes32 position) public returns (bool) {
```

Comment:
maybe we shouldn't use starting underscores for the public functions

Reply:
fixed